src/python/buildBinaryLineDocs.py

```diff
@@ -49,12 +49,12 @@ def flush(pending, pendingDocCount, fOut):
     totalLength = len(titleBytes)+len(bodyBytes)+len(randomLabelBytes)+20
     #print('len=%s' % totalLength)
     #print('HERE: %s, offset=%s' % (struct.pack('i', totalLength), fOut.tell()))
-    pending.write(struct.pack('iiiil', len(titleBytes), len(bodyBytes), len(randomLabelBytes), timeSec, msecSinceEpoch))
+    pending.write(struct.pack('iiiiil', len(titleBytes), len(bodyBytes), len(randomLabelBytes), timeSec, msecSinceEpoch))
```

Comment:
Did you maybe forget to remove one `i` from the format passed to `struct.pack` here?

Reply:
Ahhh my bad!